Cargo.toml

```diff
@@ -122,6 +122,8 @@ tendermint-proto = { version = "0.31" }
 
 # Using the IPC SDK without the `fil-actor` feature so as not to depend on the actor `Runtime`.
 ipc-sdk = { git = "https://github.com/consensus-shipyard/ipc-actors.git", default-features = false, branch = "main" }
+ipc-agent = { git = "https://github.com/consensus-shipyard/ipc-agent.git", branch = "main" }
+ipc-agent-sdk = { git = "https://github.com/consensus-shipyard/ipc-agent.git", branch = "dev" }
```

Comment:
We should probably use the same branch for anything coming from the same repo, no?

Reply:
I accidentally committed the experiment, it's updated. But now with the new `ipc-agent` dev, everything broke again.